spyder/plugins/breakpoints/widgets/breakpointsgui.py

```diff
@@ -184,35 +184,29 @@ def contextMenuEvent(self, event):
         index_clicked = self.indexAt(event.pos())
         actions = []
         self.popup_menu = QMenu(self)
-        clear_all_breakpoints_action = create_action(self,
-            _("Clear breakpoints in all files"),
+        clear_all_breakpoints_action = create_action(
+            self, _("Clear breakpoints in all files"),
             triggered=lambda: self.clear_all_breakpoints.emit())
         actions.append(clear_all_breakpoints_action)
         if self.model.breakpoints:
             c_row = index_clicked.row()
             filename = self.model.breakpoints[c_row][COL_FULL]
             lineno = int(self.model.breakpoints[c_row][COL_LINE])
-            # QAction.triggered works differently for PySide and PyQt
-            if not API == 'pyside':
-                clear_slot = lambda _checked, filename=filename, lineno=lineno: \
-                    self.clear_breakpoint.emit(filename, lineno)
-                edit_slot = lambda _checked, filename=filename, lineno=lineno: \
-                    (self.edit_goto.emit(filename, lineno, ''),
-                     self.set_or_edit_conditional_breakpoint.emit())
-            else:
-                clear_slot = lambda filename=filename, lineno=lineno: \
-                    self.clear_breakpoint.emit(filename, lineno)
-                edit_slot = lambda filename=filename, lineno=lineno: \
-                    (self.edit_goto.emit(filename, lineno, ''),
-                     self.set_or_edit_conditional_breakpoint.emit())
-
-            clear_breakpoint_action = create_action(self,
-                    _("Clear this breakpoint"),
+
+            def clear_slot(filename=filename, lineno=lineno):

```

Comment:
Did you try this manually to see if it's working as before? The thing is sometimes Qt only works with `lambda`'s, instead of proper functions.

Reply:
Yes, I try all the functions before pushing them to the PR.

Reply:
They are working as expected!